setup.scripts/spamassassin/mailwatch-spamassassin.inc

```diff
@@ -41,7 +45,7 @@ function install-spamassassin() {
 function install-bayesSql() {
     if [ "$installBayesSql" != "n" ]; then
         logprint "Configure Bayes-Sql"
-        mysql -u "$SqlRoot" -p"$SqlRootPwd" -h "$SqlHost" "$SqlDb" < "$InstallFilesFolder/setup.scripts/spamassassin/bayesSql_mysql.sql"
+        mysql -u "$SqlUser" -p"$SqlPass" -h "$SqlHost" "$SqlDb" < "$InstallFilesFolder/setup.scripts/spamassassin/bayes_mysql.sql"
```

Comment:
Still present:

```
Configure Bayes-Sql
Enter password:
```

Unless when we run setup.sh for the first time.

Can we have:

`Enter password for mailwatch: (Input will not be shown)`